File: submissions/abc/115/A.js
'use strict'

const main = (input) => {
    const d = parseInt(input)

    if(d === 25){
        console.log('Christmas')
        return 
    }
    if(d === 24){
        console.log('Christmas Eve')
        return 
    }
    if(d === 23){
        console.log('Christmas Eve Eve')
        return 
    }
    if(d === 22){
        console.log('Christmas Eve Eve Eve')
        return 
    }
}

main(require('fs').readFileSync('/dev/stdin', 'utf-8'))

File: submissions/abc/115/A.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{
    int d;
    cin >> d;
    string s;
    if(d == 25){
        s = "Christmas";
    }else if(d == 24){
        s = "Christmas Eve";
    }else if(d == 23){
        s = "Christmas Eve Eve";
    }else if(d == 22){
        s = "Christmas Eve Eve Eve";
    }
    cout << s << endl;
    return 0;
}